src/test/java/org/cyclops/integrateddynamics/core/evaluate/variable/TestParseOperators.java

```
@@ -0,0 +1,534 @@
+package org.cyclops.integrateddynamics.core.evaluate.variable;
+
+import org.cyclops.integrateddynamics.api.evaluate.EvaluationException;
+import org.cyclops.integrateddynamics.api.evaluate.variable.IValue;
+import org.cyclops.integrateddynamics.api.evaluate.variable.IVariable;
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.hamcrest.CoreMatchers.instanceOf;
+import static org.hamcrest.CoreMatchers.is;
+import static org.junit.Assert.assertThat;
+
+import static org.cyclops.integrateddynamics.core.evaluate.variable.ValueParseOperators.*;
+
+/**
+ * Test the different integer operators.
+ * @author rubensworks
+ */
+public class TestParseOperators {
+
+    private ValueParseOperators vpo = new ValueParseOperators();
+
+    private static IVariable[] s(String v){
+        IVariable[] ret = new IVariable[1];
+        ret[0] = new DummyVariableString(ValueTypeString.ValueString.of(v));
+        return ret;
+    }
+
+    @Before
+    public void before() {
+        ValueParseOperators.load();
+    }
+
+    /**
+     * ----------------------------------- INTEGER -----------------------------------
+     */
+    @Test
```

Invalid values should throw an error.